packages/material-ui/src/TextField/TextField.js

```diff
@@ -156,6 +166,11 @@ class TextField extends React.Component {
         ) : (
           InputElement
         )}
+        {characterLimit && (
+          <p style={styles.characterLimit}>
+            {value.length}/{characterLimit}
+          </p>
+        )}
```

Comment:
A `Typography` variant might be better suited I suppose. This doesn't look like the mocks in the spec.

Reply:
Understood, let me make the changes.